scripts/tests/ingest-fp.test.sh

```
@@ -0,0 +1,370 @@
+# --- 規定側: 版タグによる識別と、承認の終端化（CodeRabbit 指摘 1・2） ---
+has "識別: マーカーの版付き書式が規定にある" "$MIGRATION_DOC" "fp:2:"
+has "識別: 版タグの不在＝旧形式と定義している" "$MIGRATION_DOC" "版タグが無いものが旧形式"
+has "識別: 推論ではなく識別だと明記している" "$MIGRATION_DOC" "推論しない"
+has "識別: 混在が正常であると明記している" "$MIGRATION_DOC" "混在"
+has "識別: 検出式が規定に載っている" "$MIGRATION_DOC" "fp:[0-9a-f]{12}"
+has "終端: 承認済という終端分類がある" "$MIGRATION_DOC" "承認済"
+has "終端: 承認の効果が版タグ付きの値を書くことだと定義されている" "$MIGRATION_DOC" \
+    "承認の効果"
+has "終端: 取得不能は承認で終端できないと明記している" "$MIGRATION_DOC" \
+    '`取得不能` は承認で終端できない'
+has "終端: 完了条件が終端分類の語彙で書かれている" "$MIGRATION_DOC" '`移行済` または `承認済'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)